stack_test.go

```diff
@@ -0,0 +1,22 @@
+package slogx_test
+
+import (
+	"testing"
+
+	"github.com/powerman/check"
+
+	"github.com/powerman/slogx"
+)
+
+func TestStack(tt *testing.T) {
+	t := check.T(tt)
+
+	stack := slogx.Stack()
+	t.DeepEqual(stack.Key, slogx.KeyStack)
+	t.Match(stack.Value, "github.com/powerman/slogx_test.TestStack")
+	t.Match(stack.Value, "/slogx/stack_test.go:14")
+	t.Match(stack.Value, "testing.tRunner")
+	t.Match(stack.Value, "/testing/testing.go:1595")
```

Comment:
As I already mentioned, line numbers in testing package may change with every Go release, which will break this test. All you need here is to check for few things (actually - all things which **our** code does):
- stack.Value begins with usual "…goroutine X" prefix (i.e. we've correctly moved line1)
- first function in stack.Value is `"/slogx/stack_test.go:14"` (i.e. we've correctly removed line2-3)
- stack.Value does not ends with `"\n"`